[CDS-83440]: Update download-and-copy-artifacts-using-the-command-step.md (#5187)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/cd-steps/utilities/download-and-copy-artifacts-using-the-command-step.md

![](../cd-general-steps/static/download-and-copy-artifacts-using-the-command-step-10.png)

:::note

> Since download artifact will prepare the script to be run on host instance using `curl` in case of `Bash` and `Invoke-WebRequest` for `PowerShell`, it is recomended that the configured artifact file path contain no spaces. 

Reviewer: ```suggestion
Downloading artifact prepares the script to run on host instances using `curl1. In case of `Bash` and `Invoke-WebRequest` for `PowerShell`, we recommend that the configured artifact file path does not contain any spaces. 
```
